runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/DoFnOperator.java

```diff
@@ -918,6 +922,16 @@ public TimerInternals timerInternals() {
 
   class FlinkTimerInternals implements TimerInternals {
 
+    /** Pending timers which are necessary for supporting removal of existing timers. */
```

Comment:
What are "pending timers"?

Reply:
I'll clarify. Basically timers which have not been fired yet.